src/test/java/org/openlmis/servicedesk/service/IssueServiceTest.java

```diff
@@ -65,10 +64,12 @@
 import org.openlmis.servicedesk.web.issue.IssueDtoDataBuilder;
 import org.springframework.http.ResponseEntity;
 import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.util.ReflectionTestUtils;
 
 @RunWith(MockitoJUnitRunner.class)
 public class IssueServiceTest {
 
+  private static final String userEmail = "user@siglofa.com";
```

Comment:
static final field names are in uppercase. Moreover private fields should be at the bottom (below public)

Reply:
add change field name. User email indicates that this is the same email than user field, however it's different. So i think it's should be named rather ADMIN_EMAIL

Reply:
Fixed